src/modules/EventList/index.tsx

```diff
@@ -728,6 +817,37 @@ export const EventList = ({ useConfigOptions = false }: EventListProps): ReactEl
         },
     });
 
+    const userQuery = useQuery<EventListCurrentUserQuery>({
+        requestPolicy: 'cache-only',
+        query: gql`
+            query EventListCurrentUser {
+                currentUser {
+                    id
+                }
+            }
+        `,
+    });
+
+    const filterByTypeOptions: FilterByTypeOption[] = useMemo(() => {
```

Comment:
Instead of hardcoding the filters in the UI above, make them a memoized const since these are config-driven now